### ISSUE TITLE
Newest Release | Invalid Certificate

### ISSUE THREAD

**Issue by Xenox003** [rgb-master-team/RGBMaster#57]
Hi there, i just tried to install the latest version of RGBMaster and i noticed i am getting an error when installing:
Add-AppPackage | 0x80073CF0

after further investigation i found out that this error is telling that the certificate is invalid or not installed,.
When checking the certificate in the Windows Cert Manager i noticed it is valid until the 18. of October 2022, which makes it invalid as of now.

**Comment by Xenox003** [comment]
![image](https://user-images.githubusercontent.com/68594244/197827391-b4d1c785-8cf1-4ce9-9392-2cf2a557e477.png)
![image](https://user-images.githubusercontent.com/68594244/197827566-cc59e641-ab66-48c3-a5ac-f0a7d63a47a0.png)

**Comment by TheTrueMysiak** [comment]
Stupid workaround but if you set your windows time back wouldn't it work ?

**Comment by Xenox003** [comment]
Sure thats a workaround, i don't wanna re-configure my system time and date just for one application, which is probably understandable

**Comment by TheTrueMysiak** [comment]
After install you should be able to change it back

**Comment by Xenox003** [comment]
Let me actually test that rq, didn't think about that tbh

*Edit*
It actually worked that way, its still a good thing to replace/renew the certificate

**Comment by AspireOne** [comment]
Yep, JUST ran into it. What a terrible timing; have I bought the Yeelight a few days earlier, it would have worked :D

@Xenon003 I confirm that changing your system time works.

**Comment by TheTrueMysiak** [comment]
> Let me actually test that rq, didn't think about that tbh
> 
> _Edit_ It actually worked that way, its still a good thing to replace/renew the certificate

Yes but from what I know this project hasn't been touched in some time

**Comment by deankevorkian** [comment]
Hey @Xenox003 thank you for opening the issue, and @TheTrueMysiak thanks for the aid!
As you stated the project hasn't been touched in some time. That is not to say it is discontinued, but I am struggling to find enough time to continue developing it, being very busy with my own endeavors. Because of the frameworks being used in this project were, and have remained for the time being, unstable and difficult to use, it has greatly contributed to declining the development pace..

I will make sure to the renew the certificate soon enough and release a version with that renewed certificate. Needless to say (unrelated to this) I welcome contributions that can help bring the app to good maintenance and support issues or inconsistencies.

Really happy to see people still sticking around time to time helping issues so thanks again and I'll be sure to renew the certificate as mentioned. Keeping this open for now
